app/views/posts/_write.haml

```diff
@@ -11,8 +11,22 @@
     #access_list
       = f.label :viewer_ids, '&#8627; Who can view this post? '.html_safe
       = f.collection_select(:viewer_ids, User.where.not(id: post.user_id).order('username asc'),
-      :id, :username, {selected: post.viewer_ids}, {multiple: true})
-    %br
+      :id, :username, {selected: @viewer_ids.nil? ? post.viewer_ids : @viewer_ids}, {multiple: true})
+
+    - if post.persisted?
+      #current_authors
+        = f.label :joined_author_ids, 'Current authors:'
+        - authors = post.joined_authors.order('LOWER(users.username)')
+        = authors.map { |author| link_to author.username, user_path(author) }.join(', ').html_safe
```

Comment:
Were we going to have `×` buttons to delete authors from the thread? I forget what we decided on